api/src/main/java/com/okta/authn/sdk/client/AuthenticationClient.java

```diff
@@ -215,14 +215,14 @@ default AuthenticationResponse resetPassword(ChangePasswordRequest changePasswor
      * @see <a href="https://developer.okta.com/docs/api/resources/factors#supported-factors-for-providers">supported factor profiles</a>
      * @param factorType type of factor
      * @param factorProvider factor provider
-     * @param factorProfile profile of a supported factor
+     * @param userFactorProfile profile of a supported user factor
      * @param stateToken state token for current transaction
      * @param stateHandler State handler that handles the resulting status change corresponding to the Okta authentication state machine
      * @return An authentication response
      * @throws AuthenticationException any other authentication related error
      */
     @ApiReference(path = "/api/v1/authn/factors", href = "https://developer.okta.com/docs/api/resources/authn#enroll-factor")
-    AuthenticationResponse enrollFactor(FactorType factorType, FactorProvider factorProvider, FactorProfile factorProfile, String stateToken, AuthenticationStateHandler stateHandler) throws AuthenticationException;
+    AuthenticationResponse enrollFactor(FactorType factorType, FactorProvider factorProvider, UserFactorProfile userFactorProfile, String stateToken, AuthenticationStateHandler stateHandler) throws AuthenticationException;
```

Comment:
Consider naming this "Factor" instead of the "profile" options.  Or keep it as "FactorProfile", no need to add the "User" prefix (IMHO)

Reply:
Sure, will keep it `FactorProfile`.

Reply:
Done.